src/pem/_core.py

```diff
@@ -184,6 +184,24 @@ class OpenSSHPrivateKey(PrivateKey):
     """
 
 
+class SSHPublicKey(PublicKey):
+    """
+    A public key in SSH RFC47716 format .
+
+    The Secure Shell (SSH) Public Key File Format.
+
+    .. versionadded:: 21.1.0
+    """
+
+
+class SSHCOMPrivateKey(PrivateKey):
```

Comment:
Would you agree, that `SSHdotComPrivateKey` reads better and conveys the meaning better?

Reply:
PuTTYget refer to this format as `sshcom` 
I have not found `SSHdotCom` in use over the internet.

I prefer SSHCOMPrivateKey to keep it similar to Putty and don't invent another name.

SSH.com is now named Tectia... but I sttill prefer SSHCOMPrivateKey over TectiaPrivateKey